data-non-standard.js

```diff
@@ -1218,7 +1212,7 @@ exports.tests = [
     firefox12: false,
     firefox16: true,
     firefox17: false,
-    firefox20: true,
+    firefox20: false,
     firefox28: false,
```

Comment:
I see now what you mean.  I tried to remove redundant data in a previous patch, and had to revert it because I was throwing data away.  Apparently having these 2 redundant lines mean that both browsers were actually tested.  There's a lot of this in this file.  I kept both because I didn't wat to have my patch refused again because of the optimization.

These browsers ARE rendered in this table if you check the "Obsolete Browsers" checkbox.  For the other tables, the data is there and is wrong, so I believe it is worth fixing even if they are not displayed by the default renderer.